frontend/src/reducers/Comment.js

```diff
@@ -4,6 +4,28 @@ const initialState = {
 
 export default function comment(state=initialState, action) {
 	switch (action.type) {
+		case "GET_COMMENT:
```

Comment:
I don't think we will really need all these functions. We can just edit database from backend, and then re-render